test/roles.test.js

```diff
@@ -0,0 +1,79 @@
+// import { stub, assert } from 'sinon';
+import chai, { request, expect } from 'chai';
+import chaiHttp from 'chai-http';
+import { config } from 'dotenv';
+import app from '../src/app';
+import { saveUser } from './mocks/index';
+import { findIdUser } from '../src/services/RolesServices';
+import adminScript from '../src/config/init';
+
+config();
+
+chai.use(chaiHttp);
+
+let superToken;
+
+let id;
+
+describe('ROLE END-POINT TEST', async () => {
+  await adminScript();
+  describe('ASSIGNROLE USER TEST', () => {
+    before(async () => {
+      await request(app).post('/api/auth/register').send(saveUser);
+
```

Comment:
@itisWasp **Important Notice**
This test suite isn't being run, try removing `async` in the describe option and you can use a single describe in this case. Also, is there a particular reason you're not calling the `adminScript` in the before hook?